deployments/qat_plugin/qat_plugin.yaml

```diff
@@ -38,7 +38,7 @@ spec:
               name: intel-qat-plugin-config
               key: DEBUG
         imagePullPolicy: IfNotPresent
-        command: ["/usr/bin/intel_qat_device_plugin", "-dpdk-driver", "$(DPDK_DRIVER)", "-kernel-vf-drivers", "$(KERNEL_VF_DRIVERS)", "-max-num-devices", "$(MAX_NUM_DEVICES)", "-debug", "$(DEBUG)"]
+        command: ["/usr/local/bin/intel_qat_device_plugin", "-dpdk-driver", "$(DPDK_DRIVER)", "-kernel-vf-drivers", "$(KERNEL_VF_DRIVERS)", "-max-num-devices", "$(MAX_NUM_DEVICES)", "-debug", "$(DEBUG)"]
```

Comment:
can you check what @bart0sh did in other images in regards of `CMD` and `ENTRYPOINT` and update QAT related images?